contracts/infrastructure/dapp/CompoundCTokenFilter.sol

```diff
@@ -37,8 +43,21 @@ contract CompoundCTokenFilter is BaseFilter {
         bytes4 method = getMethod(_data);
         // cToken methods
         if (_spender == _to) {
-            // block repayBorrowBehalf
-            return (method != CTOKEN_REPAY_BORROW_BEHALF);
+            if (underlying == address(0)) {
```

Comment:
Shouldn't this address be checked for `0xEeeeeEeeeEeEeeEeEeEeeEEEeeeeEeeeeeeeEEeE`?

Reply:
Same for line 41 above.

Reply:
We have moved to `0x00...00` for the ETH address in the new contracts but you are right that we still use `0xEe...eE` in the configuration.

I suggest to just update the config file for consistency?

Reply:
okay I've updated the test config with the 0 address